docs/standard/system/generating-statements.rst

```diff
@@ -16,12 +16,12 @@ The schema enforces a minimum statement identifier length of 32 characters, and
 
 Statement identifiers are generally for internal use within applications. In most circumstances they do not need to be displayed to users. This is in contrast to :ref:`entity or person identifiers <guidance-identifiers>`, which are useful to display to users. 
 
-Publishers MAY use one of the following strategies for creating statement identifiers.
-
 Strategies for identifier creation
 ----------------------------------
 
-* Generate a `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ for the first Statement pertaining to a record. Store this in the data management system. Update it whenever the relevant record is updated. 
+Publishers MAY use one of the following strategies to create statement identifiers.
+
+* Generate a `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ for the first Statement related to a record. Store this in the data management system. Update it whenever the relevant record is updated. 
```

Comment:
```suggestion
* Generate a `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ for the first Statement for a record. Store this in the data management system. Update it whenever the relevant record is updated. 
```
Maybe avoid the word 'related' in this context since it could mean a more attenuated link and it also resonates with the idea of a relationship statement?